packages/js/components/src/experimental-select-tree-control/stories/index.tsx

```diff
@@ -28,35 +28,37 @@ const listItems: Item[] = [
 	{ id: '3', name: 'Other' },
 ];
 
+const getFilteredItems = ( items, searchValue ) => {
```

Comment:
This is in theory the same thing as having it inside the SelectTree. So if you are not going to remove this function in this PR then we should mark the function as something to be removed in the future.